stuf_zds_payments/apps.py

```diff
@@ -1,5 +1,11 @@
 from django.apps import AppConfig
+from django.utils.translation import gettext_lazy as _
 
 
 class StufZdsPaymentsConfig(AppConfig):
     name = "stuf_zds_payments"
+    label = "stuf_zds_payments"
```

Comment:
```suggestion
    label = "registration_stuf_zds_payments"
```
to keep up with the existing pattern of properly namespacing it under the `registrations` module :)